### ISSUE TITLE
oracle support

### ISSUE THREAD

**Issue by ariva07** [rabbitmq/chef-cookbook#568]
* Cookbook version: rabbitmq 5.8.5
* OS: OracleLinux 8
* RabbitMQ version: rabbitmq-server-3.8.11-1.el7
* Erlang version provisioned: 23

After migrating to Oracle, the default recipe won't install the package properly:
```
[2021-02-10T11:43:18+01:00] ERROR: rpm_package[/var/chef/cache/] (rabbitmq::default line 224) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
---- Begin output of ["rpm", "-qp", "--queryformat", "%{NAME} %{VERSION}-%{RELEASE}\n", "/var/chef/cache/"] ----
STDOUT: 
STDERR: error: /var/chef/cache/: read failed: Is a directory (21)
```
Adding oracle to the list in recipes/default.rb, line 49 would probably easily fix the problem (tested on chef-zero)
```
 48 default_rpm_package_name = value_for_platform(
 49   %w(centos redhat scientific oracle) => {
 50     '< 7.0' => "rabbitmq-server-#{version}-1.el6.noarch.rpm",
```

**Comment by michaelklishin** [comment]
Please submit a PR.

**Comment by michaelklishin** [comment]
Should be resolved as of 3733b59.
